templates/setup/index.html.ejs

```diff
@@ -0,0 +1,88 @@
+<h1>Welcome to Solid</h1>
+<p>
+  This server implements
+  the <a href="https://solid.github.io/specification/protocol">Solid protocol</a>
+  so you can create your own <a href="https://solidproject.org/about">Solid Pod</a>
+  and identity.
+</p>
+
+<h2 id="public">Making this server public</h2>
+<p>
+  Before making this server public,
+  you might want to <strong>disable Pod registration</strong>
+  by <a href="https://github.com/solid/community-server/blob/main/config/identity/README.md">changing
+    the configuration</a>.
+</p>
```

Comment:
This text is duplicated from somewhere else; can/should we fix that?
(or at least have a comment to cross-reference them)

Reply:
I copied this from the `index.html` that you wrote so I'll leave it up to you. The EJS templates now support partials so we can do something with that.